### DISCUSSION TITLE
Welcome to AIStoryBuildersOnline Discussions!

### DISCUSSION THREAD

**Discussion by ADefWebserver** [AIStoryBuilders/AIStoryBuildersOnline#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikethediyman** [comment]
Hi there I'm a 75 year old retired salesman living in Ireland with my wife Catherine, we live in a quiet village not far from the capitol Dublin and my son and his family also live in our neighbourhood which is great as we see our 2 grandchildren Molly and Louisa very often !
I have always wanted to write a book and have quite a few ideas but get a lot of " writers block " so once AI came along I decided to use it to assist me along the way. Since doing some research online I have found quite a lot of very useful resources and advice from fellow writers and I was especially delighted when I came across AIStoryBuilders and can't wait to get started with it. Kind regards Michael

**Comment by ariffinsetya** [comment]
Hi I'm ariffinsetya, a 30-something years old LLM Hobbyist. Love stories, and dabble with LLM to write one, for role play and just write any stories in any scenarios. I'm curious if it's possible to add option for OpenAI Compatible for LLM Settings, as currently there's only OpenAI and Azure OpenAI. I especially interested to use local hosted OpenAI Compatible LLM, since I just prefer to use my own hardware, as it's cheaper and more safe that way. Best regards, ariffinsetya

**Comment by ADefWebserver** [comment]
I plan to offer this in January. I won't have time to work on it before
then.

Thank You

On Thu, Nov 28, 2024 at 7:46 PM ariffinsetya ***@***.***>
wrote:

> Hi I'm ariffinsetya, a 30-something years old LLM Hobbyist. Love stories,
> and dabble with LLM to write one, for role play and just write any stories
> in any scenarios. I'm curious if it's possible to add option for OpenAI
> Compatible for LLM Settings, as currently there's only OpenAI and Azure
> OpenAI. I especially interested to use local hosted OpenAI Compatible LLM,
> since I just prefer to use my own hardware, as it's cheaper and more safe
> that way. Best regards, ariffinsetya
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AIStoryBuilders/AIStoryBuildersOnline/discussions/1#discussioncomment-11411865>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAOFSB4PRGB4WCVV6S7ZU532C7PSHAVCNFSM6AAAAABSWH5LGSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNBRGE4DMNI>
> .
> You are receiving this because you authored the thread.Message ID:
> <AIStoryBuilders/AIStoryBuildersOnline/repo-discussions/1/comments/11411865
> @github.com>
>


-- 
Michael Washington
***@***.***

http://ADefWebserver.com
http://BlazorHelpWebsite.com
http://AIStoryBuilders.com
